
package cl.c2c.ejbca;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.5.6
 * 2021-02-10T23:01:14.699-03:00
 * Generated source version: 2.5.6
 */

@WebFault(name = "UserDataSourceException", targetNamespace = "http://ws.protocol.core.ejbca.org/")
public class UserDataSourceException_Exception extends Exception {
    
    private cl.c2c.ejbca.UserDataSourceException userDataSourceException;

    public UserDataSourceException_Exception() {
        super();
    }
    
    public UserDataSourceException_Exception(String message) {
        super(message);
    }
    
    public UserDataSourceException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public UserDataSourceException_Exception(String message, cl.c2c.ejbca.UserDataSourceException userDataSourceException) {
        super(message);
        this.userDataSourceException = userDataSourceException;
    }

    public UserDataSourceException_Exception(String message, cl.c2c.ejbca.UserDataSourceException userDataSourceException, Throwable cause) {
        super(message, cause);
        this.userDataSourceException = userDataSourceException;
    }

    public cl.c2c.ejbca.UserDataSourceException getFaultInfo() {
        return this.userDataSourceException;
    }
}
